ChangeLog.md

```diff
@@ -191,6 +191,10 @@
         - New-AzApplicationGatewayRewriteRuleCondition
     - Cmdlets updated with optional parameter - RuleSequence and Condition
         - New-AzApplicationGatewayRewriteRule
+* Added support for updating tags in Set operations for VirtualNetworkGateways and VirtualNetworkGatewayConnections
```

Comment:
There shouldn't be any changes here

Reply:
This PR contains only alredy approved PRs by PowerShell team... @MiYanni for visibility.
Not sure why can't we add changes to changeLog?